app/facades/food_facade.rb

```diff
@@ -0,0 +1,11 @@
+class FoodFacade
+
+  def self.find_food_results(food_item)
+    data = FoodService.search_foods(food_item)
+    total_count = data[:totalHits]
+    info = data[:foods].map do |food_info|
+      Food.new(food_info)
+    end
+    [total_count, info ]
+  end
```

Comment:
👍 Great work here! I like that you are grabbing all the info you need from this one API call.